go-fuzz/main.go

```diff
@@ -42,6 +42,7 @@ var (
 	flagSonar             = flag.Bool("sonar", true, "use sonar hints")
 	flagV                 = flag.Int("v", 0, "verbosity level")
 	flagHTTP              = flag.String("http", "", "HTTP server listen address (coordinator mode only)")
+	flagDict              = flag.String("dict", "", "Dictionary file containing string tokens (one per line, not quoted)")
```

Comment:
All other flag descriptions start with a non-capital letter, please use the same style.

Reply:
Ahh, I only checked HTTP above... but that's an acronym 😅